backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ActionDefinitionImpl.java

```diff
@@ -68,14 +68,18 @@ protected EClass eStaticClass() {
      */
     @Override
     public EList<Feature> getParameter() {
-        List<Feature> features = this.getOwnedRelationship().stream()
-                .filter(ParameterMembership.class::isInstance)
-                .map(ParameterMembership.class::cast)
-                .flatMap(or -> or.getOwnedRelatedElement().stream())
-                .filter(Feature.class::isInstance)
-                .map(Feature.class::cast)
+        List<Feature> data = this.getOwnedFeature().stream()
```

Comment:
I uniformized the implementation of `getParameter` with `Step#getParameter`. I didn't find a reason for this implementation in the specification.